app/models/chf/citable_attributes.rb

```diff
@@ -35,9 +35,16 @@ def initialize(work,
       @collection = collection.nil? ? work.in_collection_presenters.first : collection
       @parent_work = parent_work.nil? ? work.parent_work_presenters.first : parent_work
       @edge_case_date_literals = !!edge_case_date_literals
-      @implementation = treat_as_local_photograph? ?
-        TreatAsLocalPhotograph.new(@work, collection: @collection, parent_work: @parent_work) :
-        StandardTreatment.new(@work, collection: @collection, parent_work: @parent_work)
+
+      if work.genre_string != nil && work.genre_string.include?('Oral histories')
```

Comment:
Extract this into a method to parallel the `treat_as_local_photograph?` line? Maybe `cite_as_oral_history?` ?